ui/src/app.tsx

```diff
@@ -116,7 +128,7 @@ const
                 const page = e.page
                 const lightbox = lightboxB()
                 return (
-                  <Fluent.ThemeProvider>
+                  <Fluent.ThemeProvider theme={{ components: componentCustomStyles }}>
```

Comment:
Use a file-scoped const instead to improve perf as suggested in Fluent docs:

<img width="1029" alt="image" src="https://user-images.githubusercontent.com/64769322/203485523-25ee2a7e-7f3c-40ef-94a0-a0466442ff4a.png">

Reply:
I've first moved the constant `componentCustomStyles`  from inside `bond()` to the upper scope, but now I see I should move also whole `{components: componentCustomStyles}` object. 
Thank you @mturoci , now it should be correct.